cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/store/remote/RemoteOpsClient.java

```diff
@@ -73,6 +89,24 @@ private String resolve(String resource) {
                          addr.getHostName(), addr.getPort(), "/v1", resource);
   }
 
+
+  protected void executeRequest(String methodName, Object... arguments) {
+    doPost("execute/" + methodName, GSON.toJson(createArguments(arguments)));
+  }
+
+  private static List<MethodArgument> createArguments(Object... arguments) {
```

Comment:
nit: do we need this method? only seems to be called once. Also, if needed, does it need to be static?